app/views/entitlements/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<%- @page_title = "#{@entitlement.rank} Entitlements" -%>
```

Comment:
You could make `entitlement` a helper in the controller, then omit calling `entitlement` inside `show`. Then just use `entitlement` directly in the template.

Alternatively, I would drop the method out of the controller, and just assign the variable in the show action.